src/splitseq_mkref.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
```

Comment:
Consider adding `set -euxo pipefail` for a more robust script. This might break something, but shouldn't be too hard to fix. It's kinda like Perl's strict mode. (See https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/ for some explanation).

Reply:
Should I add those options to line 17 where `set -eu` currently is? I put it there to permit the mkdir of the outer directory to complain that it already exists.  Alternatively, I could put that mkdir inside an if statement.

Reply:
I just edited the existing line.  Let me know after I push if you'd prefer I add a conditional so it can be at the top.